inc/class-bento-events-controller.php

```
@@ -44,12 +44,15 @@ public static function get_bento_option( $option_name ) {
-		protected static function enqueue_event( $user_id, $type, $email, $details = array() ) {
-			if ( empty( self::get_bento_option( 'bento_events_recurrence' ) ) ) {
-				delete_option( self::EVENTS_QUEUE_OPTION_KEY );
-				return;
-			}
-
+	protected static function enqueue_event( $user_id, $type, $email, $details = array() ) {
+		// Always enqueue events regardless of recurrence setting
+		// Events will be processed when cron runs or manually triggered
+		
+		$sanitized_email = self::sanitize_email_for_logging( $email );
+		$sanitized_details = self::sanitize_details_for_logging( $details );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sanitized_details'. (Unused Code Rules)

(UnusedLocalVariable)